src/OhadSoft.AzureLetsEncrypt.Renewal/Management/AzureEnvironmentParams.cs

```diff
@@ -48,11 +53,6 @@ private T ValidateParam<T>(T value, string name, Func<T, string, T> validator)
             return validator((T)nullable, name);
         }
 
-        public override string ToString()
```

Comment:
are you sure this isn't printed out somewhere?

Reply:
resolving here per below comment as well